block/manager.go

```diff
@@ -150,23 +149,27 @@ func (m *Manager) Start(ctx context.Context) error {
 	}
 
 	if !isSequencer {
+		// Fullnode loop can start before syncing from DA
 		go uevent.MustSubscribe(ctx, m.Pubsub, "applyGossipedBlocksLoop", p2p.EventQueryNewNewGossipedBlock, m.onNewGossipedBlock, m.logger)
 	}
 
+	// TODO: populate the accumulatedSize on startup
+
 	err = m.syncBlockManager()
 	if err != nil {
 		return fmt.Errorf("sync block manager: %w", err)
 	}
 
+	// Sequencer producer loop and full-node retrieve loop is started only when finished syncing from DA
```

Comment:
the comment here seems incorrect in regards to the full node

Reply:
that was supposed to be removed. fixed